thegraph-core/src/types/subgraph_id.rs

```diff
@@ -76,12 +76,32 @@ impl std::str::FromStr for SubgraphId {
 }
 
 impl std::fmt::Display for SubgraphId {
+    /// Format the `SubgraphId` as a base58-encoded string.
+    ///
+    /// ```rust
+    /// use thegraph_core::subgraph_id;
+    /// use thegraph_core::types::SubgraphId;
+    ///
+    /// const ID: SubgraphId = subgraph_id!("DZz4kDTdmzWLWsV373w2bSmoar3umKKH9y82SUKr5qmp");
+    ///
+    /// assert_eq!(format!("{}", ID), "DZz4kDTdmzWLWsV373w2bSmoar3umKKH9y82SUKr5qmp");
+    /// ```
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         f.write_str(&bs58::encode(self.0.as_slice()).into_string())
     }
 }
 
 impl std::fmt::Debug for SubgraphId {
+    /// Format the `SubgraphId` as a debug string.
+    ///
+    /// ```rust
+    /// use thegraph_core::subgraph_id;
+    /// use thegraph_core::types::SubgraphId;
+    ///
+    /// const ID: SubgraphId = subgraph_id!("DZz4kDTdmzWLWsV373w2bSmoar3umKKH9y82SUKr5qmp");
+    ///
+    /// assert_eq!(format!("{:?}", ID), "Subgraph(DZz4kDTdmzWLWsV373w2bSmoar3umKKH9y82SUKr5qmp)");
+    /// ```
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         write!(f, "Subgraph({})", self)
```

Comment:
Seems like this should be `SubgraphId` to be consistent with `DeploymentId`
```suggestion
    /// assert_eq!(format!("{:?}", ID), "SubgraphId(DZz4kDTdmzWLWsV373w2bSmoar3umKKH9y82SUKr5qmp)");
    /// ```
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        write!(f, "SubgraphId({})", self)
```

Reply:
Good catch! 👀